packages/rolldown/tests/fixtures/builtin-plugin/transform/main.js.snap

```diff
@@ -0,0 +1,12 @@
+Error: Build failed with 1 error:
+
+[UNHANDLEABLE_ERROR] Error: Something went wrong inside rolldown, please report this problem at https://github.com/rolldown/rolldown/issues.
+
+[builtin:transform] Error: Namespace tags are not supported by default. React's JSX doesn't support namespace tags. You can set `throwIfNamespace: false` to bypass this warning.
+   ╭─[ D:\shulaoda\rolldown\packages\rolldown\tests\fixtures\builtin-plugin\transform\jsx.abc:1:19 ]
```

Comment:
The test will not passed , you should provide a relative path(stable_id in rolldown)